### ISSUE TITLE
Invalid dependency generated by split components (v4.16.11 refactoring)

### ISSUE THREAD

**Issue by Piezoid** [marko-js/marko#1320]
### Marko Version: 4.16.11 
v4.16.10 doesn't have this problem.

### Details
Split components generate a lasso error while resolving a non existent `marko/src/components` JS module:
`Error: Failed to resolve "marko/src/components". Target file does not exist. Started search from directory "/tmp/marko_app/src/components/site-layout"`

the `.marko.js` contains:
```js
    deps: [
      "./style.css",
      "marko/src/components"
    ],
```

Worked fine with v4.16.10, before the renaming `marko/components` -> `marko/runtime/components`.
Diff: [v4.16.10...v4.16.11](https://github.com/marko-js/marko/compare/v4.16.10...v4.16.11)

### Possible Fix

Tried:
`sed 's\marko/components\marko/runtime/components\'`
Found matches, but didn't solve the problem.

Also:
`grep -E "src.{1,64}components" ~/.config/yarn/global node_modules -r  | grep -v runtime`
Didn't yield interesting matches.

So no luck at finding the origin of this `"marko/src/components"`.

<details><summary>Additional Info</summary>

### Your Environment
- Arch
- node: v11.14.0
- npm: n6.9.0

### Steps to Reproduce

```bash
marko create marko_app
cd marko_app
touch src/components/site-layout/component-browser.js
npm start
```
</details>

**Comment by mlrawlings** [comment]
You should consider anything under `src`/`dist` to be a private api.  For this case, Marko has a top-level `marko/components` file you can reference (https://github.com/marko-js/marko/blob/master/components.js)

**Comment by mlrawlings** [comment]
Thanks for looking into the issue!  The fix was released in `4.16.12`.
